jekyll/_cci2/creating-orbs.md

```diff
@@ -6,39 +6,36 @@ description: "Starting point for Creating CircleCI Orbs"
 categories: [getting-started]
 order: 1
 ---
-
+## Introduction
 Orbs are reusable packages of CircleCI configuration that you may share across projects, enabling you to create encapsulated, parameterized commands, jobs, and executors that can be [used across multiple projects]({{ site.baseurl }}/2.0/using-orbs/). 
 
 [Orbs]({{ site.baseurl }}/2.0/orb-intro/) are made available for use in a configuration through the `orbs` key in the top level of your 2.1 [.circleci/config.yml]({{ site.baseurl }}/2.0/configuration-reference/) file.
 
-## Orb Publishing Process
-
-To publish an orb, follow the steps listed below as an org Admin.
-
-1. Claim a namespace (assuming you don't yet have one), using the command `circleci namespace create <NAMESPACEID> <VCS> <YOURORG>`. For example:
-`circleci namespace create sandbox github CircleCI-Public`
+### Orb Publishing Process
 
-2. Create the orb inside your namespace, for example:
-`circleci orb create sandbox/hello-world`
+Before working with orbs, you may find it helpful to gain a high-level understanding the end-to-end orbs publishing process. The diagram shown below illustrates the orbs publishing process.
+![orbsworkflow-v2 image](  {{ site.baseurl }}/assets/img/docs/orbsworkflow-v2.png)
 
-3. Create the content of your orb in a file. You will generally perform this action in your code editor in a git repo made for your orb. For example, let's assume a file in `/tmp/orb.yml` could be made with a simple orb similar to:
-`echo '{version: "2.1", description: "a sample orb"}' > /tmp/orb.yml`
+#### Steps 1 - 3 - Use the CircleCI CLI
```

Comment:
👏 I like the use of headings to break up content. They make it so much easier to skim a page.

Reply:
For this heading, `Set Up the CircleCI CLI` makes more sense to me. It's also not clear how this section is three steps anymore.

Are they numbered somewhere else as Steps 1-3? If not, this should be `Step 1 - Set Up the CircleCI CLI`

Reply:
Renumbered steps to match a new Orbs Workflow Diagram we created. Here is a link to the diagram. In the next PR, I will add a link to the image.

https://github.com/circleci/circleci-docs/blob/master/jekyll/assets/img/docs/orbsworkflow-v2_orbs_outline.png